circle.yml

```diff
@@ -0,0 +1,25 @@
+machine:
+  xcode:
+    version: "7.1"
+  environment:
+    default_destination: platform=iOS Simulator,name=iPhone 6,OS=9.0
+    XCODE_SCHEME: photocolle-sdk
+
+dependencies:
+  pre:
+    - brew update
+    - brew install ios-sim
+
+test:
+  pre:
+    # Build sdk
+    - make release
+    # Copy sdk and docs to artifacts
+    - cp target/PhotoColleSDK-iOS-*.zip $CIRCLE_ARTIFACTS
```

Comment:
It is better to separate doc and framework file to different files. Like photocolle-AndroidSDK, java doc and aar file are separate. Because it is easy to check the build results directly from circleci, not need to download.